app/assets/stylesheets/views/_history-people.scss

```diff
@@ -60,74 +58,78 @@
 
 
 .historic-people-show {
+  $gutter: 30px;
+  $grey: darken($govuk-secondary-text-colour, 2%);
+
   .person-info {
-    @include media(tablet) {
+    @include govuk-media-query($until: tablet) {
       float: left;
-      width: $one-quarter;
+      width: 25%;
     }
   }
 
   .person-info {
-
-    .name-title, .person-detail, .person-profile {
+    .name-title,
+    .person-detail,
+    .person-profile {
       .inner {
-        padding: 0 $gutter-half;
+        padding: 0 $gutter / 2;
       }
     }
 
     .person-profile {
-      @include core-14;
       width: 218px;
 
-      img {
-        max-width: $full-width;
-      }
       .info {
-        background-color: $grey-3;
-        padding-bottom: $gutter-half;
+        background-color: $grey;
+        padding-bottom: $gutter / 2;
         margin-bottom: $gutter;
-        h3, p {
-          padding: 0 $gutter-half;
+
+        h3,
+        p {
+          padding: 0 $gutter / 2;
         }
+
         h3 {
           font-weight: bold;
-          margin-top: $gutter-half;
+          margin-top: $gutter / 2;
         }
       }
     }
 
     .name-title {
       h2 {
-        @include heading-27;
         font-weight: bold;
         margin-bottom: $gutter;
+
         span {
-          @include core-19;
           display: block;
-          color: $grey-1;
-          margin-top: $gutter-one-sixth;
+          color: $grey;
+          margin-top: $gutter / 6;
         }
       }
     }
 
-    @include media(tablet) {
+    @include govuk-media-query($until: tablet) {
       position: relative;
       width: 75%;
+
       .person-profile {
         float: left;
-        width: $one-third;
-        img {
-          width: $full-width;
-        }
+        width: 33%;
+
         .inner {
-          padding: 0 $gutter-half;
+          padding: 0 $gutter / 2;
         }
       }
-      .name-title, .person-detail {
+
+      .name-title,
+      .person-detail {
```

Comment:
I'm not sure what this styling relates to (haven't looked in the heroku app yet), but we may be able to use the design system layout classes rather than adding CSS.
https://govuk-design-system-preview.netlify.app/styles/layout/

Reply:
This whole stylesheet is actually unused at the moment, it will apply to pages like https://www.gov.uk/government/history/past-foreign-secretaries/edward-wood when those get copied over to this app

Reply:
Should we remove it from this PR as it's not needed yet?